app/databaseAccessors/articleAccessor.js

```diff
@@ -328,4 +329,37 @@ export default class ArticlesAccessor {
       .exec();
     return newArticle;
   }
-}
+
+
+
+  static async deleteArticle(slug) {
+    await Connection.open();
+
+    
+
+    const article = await Article.findOne( {slug: slug} );
+    if (article) {
+
+      try {
+      console.log("we found the article");
+
+      //console.log(article);
+
+      const resp = await Article.deleteOne({ _id: article._id });
```

Comment:
If this calls the delete hook for an issue map, when is this article actually deleted from the articles list itself?